src/harbors/use-harbors.tsx

```diff
@@ -0,0 +1,76 @@
+import {useHarborsQuery} from '@atb/queries';
+import {useHarborConnectionOverrides} from '@atb/harbors/use-harbor-connection-overrides';
+import {useEffect, useState} from 'react';
+import {StopPlaceFragment} from '@atb/api/types/generated/fragments/stop-places';
+import {HarborConnectionOverrideType} from '@atb-as/config-specs';
+import {isDefined} from '@atb/utils/presence';
+import _ from 'lodash';
+
+export const useHarbors = (fromHarborId?: string) => {
+  const [isLoading, setIsLoading] = useState(true);
+  const [isSuccess, setIsSuccess] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [error, setError] = useState<unknown>();
+  const [data, setData] = useState<StopPlaceFragment[]>([]);
+
+  const harbors = useHarborsQuery();
+  const connections = useHarborsQuery(fromHarborId);
+  const overrides = useHarborConnectionOverrides(fromHarborId);
+
+  useEffect(() => {
```

Comment:
I have seen this conditions before, outside an useEffect like by example: https://github.com/AtB-AS/mittatb-app/blob/3e23d045775b8cc76a6cba92a87a7f18b08d2263/src/components/map/components/DeparturesDialogSheet.tsx#L71 In this way we can reduce the use of effects?